README.md

```diff
@@ -18,6 +18,7 @@ Self-Hosted payment processors run on your server and provide you with a full co
 
 | Processor | Fees | Lightning | Directly to Your Wallet | Conversion to Fiat | Requirements |
 | --------- |:----:|:---------:|:-----------------------:|:------------------:| ------------ |
+| [Bitcoin Opencart plugin](https://www.opencart.com/index.php?route=marketplace/extension/info&extension_id=34039) | fix | No | Yes | via forwarding | OpenCart |
```

Comment:
The plugin depends on Apirone's hosted API and can't be added as a standalone self-hosted solution. Remove it.

Reply:
You right. Thank you!